src/entities/stabilityTracker/ui/StabilityTracker.gyro.backup.js

```diff
@@ -0,0 +1,412 @@
+      PLAYGROUND_WIDTH - BLOCK_HEIGHT * 2 + OUTER_CIRCLE_RADIUS * 2,
+    );
+    userPosition.current[1] += BLOCK_HEIGHT - OUTER_CIRCLE_RADIUS;
+  };
+
+  const onUserStartedMoving = (event: GestureResponderEvent) => {
+    if (IS_TOUCH) {
+      startPosition.current = event.nativeEvent.locationY;
+    }
+  };
+
+  const onUserStoppedMoving = (event: GestureResponderEvent) => {
+    if (IS_TOUCH) {
+      if (isMoving) {
+        updateUserPosition(
+          event.nativeEvent.locationX,
+          center + event.nativeEvent.locationY - startPosition.current,
+        );
+      } else {
+        updateUserPosition(
+          event.nativeEvent.locationX,
+          event.nativeEvent.locationY,
+        );
+      }
+      startPosition.current = 0;
+    }
+    setMoving(true);
+
+    showControlBar.current = false;
+  };
+
+  const onUserMove = (event: GestureResponderEvent) => {
+    if (IS_TOUCH && !showControlBar.current) {
+      updateUserPosition(
+        event.nativeEvent.locationX,
+        center + event.nativeEvent.locationY - startPosition.current,
+      );
+    }
+  };
+
+  const finishResponse = () => {
+    onComplete(['atata']);
+
+    setMoving(false);
+    return;
+    alert('finishing response');
+    boundWasHit.current = false;
+    setNumberOfTrials(0);
+    setScore(0);
+    lambdaValue.current = INITIAL_LAMBDA;
+    circlePosition.current = CENTER_COORDINATES;
+    lambdaSlope.current = config?.lambdaSlope;
+    return;
+    return;
+
+    boundWasHit.current = false;
+
+    let maxLambda = 0;
+    for (const response of responses.current) {
+      if (maxLambda < response.lambda) {
+        maxLambda = response.lambda;
+      }
+    }
+
+    onChange(
+      {
+        maxLambda,
+        value: [...responses.current],
+        phaseType: config.phase,
+      },
+      true,
+    );
+
+    // reset values
+  };
+
+  const restartTrial = () => {
+    setScore((score * 3) / 4);
+    lambdaValue.current = lambdaValue.current / 2;
+    circlePosition.current = CENTER_COORDINATES;
+
+    numberOfTrials.current += 1;
+
+    if (IS_TRIAL) {
+      lambdaSlope.current = (lambdaSlope.current * 95) / 100;
+    }
+
+    if (numberOfTrials.current >= config.numberOfTrials && IS_TRIAL) {
+      finishResponse();
+    }
+  };
+
+  const updateScore = (deltaTime: number) => {
+    const distance = computeDistance(circlePosition.current, TARGET_POSITION);
+    const bonusMultiplier = getBonusMultiplier(
+      distance,
+      INNER_CIRCLE_RADIUS,
+      OUTER_CIRCLE_RADIUS,
+    );
+    const scoreChange = getScoreChange(bonusMultiplier, deltaTime);
+    setScore(prevScore => prevScore + scoreChange);
+  };
+
+  const updateCirclePosition = (timeElapsed: number, deltaTime: number) => {
+    const delta = computeDxDt(
+      circlePosition.current,
+      userPosition.current,
+      lambdaValue.current,
+      center,
+    );
+    const newCirclePositionY =
+      (delta[1] * deltaTime) / 1000 + circlePosition.current[1];
```

Comment:
Why there's circlePosition.current[1] over the code, can we handle somehow? Not clear what is index 1